src/smc-webapp/project.coffee

```diff
@@ -116,11 +116,16 @@ class ProjectPage
         delete @actions
         delete @store
 
+    get_total_project_quotas: (s, project_id) =>
+        quotas = s.get_total_project_quotas(project_id)
+        quotas.project_id = project_id
```

Comment:
> so, it looks as if s.get_total_project_quotas sometimes returns null or undefined?

YES -- absolutely all code **must** be written to allow for cases like this.  If the relevant project/account/etc. information isn't loaded yet from the backend, then you have to fall back to a reasonable default behavior until it is loaded.   Here, doing `quotas?.project_id = project_id` would  I think work.